### PR TITLE
Fullscreen Overlay Navigation

### PR DESCRIPTION
**Pull request by sohan9819**
Completed making the idea of full screen navbar

Here is the preview 

## Fullscreen Overlay Navigation

<h3> Tech Stack Used  </h3>
<p align="left"> <a href="https://www.w3.org/html/" target="_blank"> html5 </a> <a href="https://www.w3schools.com/css/" target="_blank"> css3 </a> <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript" target="_blank"> javascript </a>  </p>

### Preview

fullscreen_navbar

### CODE REVIEW

**Review by Abirpal202049 — APPROVED** [review]

File: Navbars/Fullscreen_Overlay_Navigation/code/css/style.css [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+  background: #fdfdea;
+  transition: all 2s ease;
+}
+h1 {
+  font-size: 4rem;
+  font-weight: 400;
+}
+h3 {
+  font-size: 1.5em;
+}
```
Comment (thepranaygupta) [inline Navbars/Fullscreen_Overlay_Navigation/code/css/style.css:20]: don't add styling directly to HTML tags, use classes instead

**Review by thepranaygupta — CHANGES_REQUESTED** [review]

File: Navbars/Fullscreen_Overlay_Navigation/code/css/style.css (hunk 1 above)
Comment (sohan9819) [reply Navbars/Fullscreen_Overlay_Navigation/code/css/style.css]: @thepranaygupta  did the changes 👍.

**Review by Abirpal202049 — APPROVED** [review]

**Review by nandini7637 — CHANGES_REQUESTED** [review]
Just add a class to body tag too else everything looks great

**Review by bhavesh-chaudhari — APPROVED** [review]

**Review by Ovenoboyo — APPROVED** [review]

**Review by thepranaygupta — APPROVED** [review]

**Comment by sohan9819** [comment]
@nandini7637  done 👍 ! added class to body

**Pull request merged**